pkg/protocol/http1/req/header_test.go

```diff
@@ -249,6 +250,18 @@ func TestRequestHeaderCookie(t *testing.T) {
 	}
 }
 
+func TestRequestHeaderCookies(t *testing.T) {
```

Comment:
This test is not a HTTP/1.1-related one. It's better to put it under pkg/protocol same with it's implementation.

Reply:
done, thx